keyboards/jidohun/qmk113/rules.mk

```diff
@@ -0,0 +1 @@
+WEAR_LEVELING_DRIVER = spi_flash
```

Comment:
```suggestion
```

Reply:
This is for selecting spi_flash driver is deleted, it does not match the actual situation and cannot be deleted